pinot-plugins/pinot-file-system/pinot-gcs/src/test/java/org/apache/pinot/plugin/filesystem/TestGcsPinotFS.java

```diff
@@ -193,6 +193,9 @@ public void testGcs()
     // Test listFiles()
     // Check that all the files are there
     assertEquals(listFilesToStream(_dataDir).collect(toSet()), expectedElements);
+    // listFiles() using only the bucket "gs://bucket" contain the same elements
+    assertTrue(
```

Comment:
Why changing it to `containsAll` instead of `equals`?

Reply:
Because the bucket used for the test might contain other directories (i.e., engineers who run the test uses a shared bucket), in which case it will contain more elements.

Reply:
I added a comment to explain this.